js/src/modal.js

```diff
@@ -556,15 +550,7 @@ EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DATA_TOGGLE, function (
     })
   })
 
-  let data = Data.get(target, DATA_KEY)
-  if (!data) {
-    const config = {
-      ...Manipulator.getDataAttributes(target),
-      ...Manipulator.getDataAttributes(this)
-    }
-
-    data = new Modal(target, config)
-  }
+  const data = Data.get(target, DATA_KEY) || new Modal(target, Manipulator.getDataAttributes(this))
```

Comment:
It is not mentioned anywhere in the docs that the data API reads the configuration from the data attributes of the toggle button also :exploding_head:

Reply:
It's not either tested. I am in favor to omit it. If you agree so. To my eyes it's not a valid behavior

Reply:
It should be omitted IMO if it is not a breaking change :thinking: